cmd/cmdtest.go

```diff
@@ -32,6 +32,7 @@ func CallCmd(
 		membersClient,
 		projectsClient,
 		"test",
+		false,
```

Comment:
I don't see NewRootCommand modified to have 7 arguments anywhere

Reply:
That should [be here](https://github.com/launchdarkly/ldcli/pull/176/files#diff-ab967ab1a2f3a1b769106eeb7bfe892ef0e81d1d27811fa15be08e6749feee1fR33).